README.md

```diff
@@ -161,7 +162,7 @@ Massive thank you in advance :heart:
 
-Thank you to David Ogles, Flobler, Jason from LDO Motors, Matthew Humphrey, Nathan Denkin, Orlando from All3DMakers, Saiz, Sleene, Termlimit, Vertigo295 for helping me developing the Bear project.
+Thank you to David Ogles, Flobler, Jason from LDO Motors, Matthew Humphrey, Nathan Denkin, Orlando from All3DMakers, Saiz, Sleene, Termlimit, vertigo295 for helping me developing the Bear project.
```

Comment:
missed a 295 here.

Reply:
Vertigo295 is 1.26 times more awesome than Vertigo235, but don't tell this to Josh ;)